### ISSUE TITLE
Feature: Add persistent escape button to touchbar

### ISSUE THREAD

**Issue by mnjongerius** [graceavery/tamagotchiTemp#19]
I keep on accidentally closing TouchbarPet because I'm used to having and escape button in that position. Would be great to have an persistent escape button ánd a TouchbarPet in my touchbar!

**Comment by n8-dev** [comment]
I think Duplicate of #10

**Comment by mnjongerius** [comment]
@n8-dev No, this is a feature request for an extra button in the interface
